panel/src/components/Forms/Writer/Toolbar.vue

```diff
@@ -22,7 +22,7 @@
 					>
 						{{ node.label }}
 					</k-dropdown-item>
-					<hr v-if="node.separator === true" :key="nodeType + '-divider'" />
+					<hr v-if="node.separator === true && nodeIndex !== Object.keys(nodeButtons).length - 1" :key="nodeType + '-divider'" />
```

Comment:
I think this would only fix the additional divider when at the end, wouldn't it?
But maybe there are other scenarios?

Wondering if hiding it via CSS could be the more reliable solution. In the direction of `hr + hr { display: none }`

Reply:
That's exactly the issue. 

> Additional divider at end of Node list when disabling nodes

I think if it is the divider and the last index, it's clearly an unnecessary divider. It adapts to all cases.

Reply:
Also I don't think `hr + hr` correctly works. This requires two consecutive hr. That's not the issue.

````
- Menu item
____________
- Menu item
- Menu item
____________ <-- the issue
````

Reply:
True